support/pgbouncer-build

```diff
@@ -2,9 +2,9 @@
 
 set -e
 
-PGBOUNCER_VERSION=${PGBOUNCER_VERSION-1.7.2}
+PGBOUNCER_VERSION=${PGBOUNCER_VERSION-1.13.0}
 
-pgbouncer_tarball_url=https://pgbouncer.github.io/downloads/files/${PGBOUNCER_VERSION}/pgbouncer-${PGBOUNCER_VERSION}.tar.gz
+pgbouncer_tarball_url=http://www.pgbouncer.org/downloads/files/${PGBOUNCER_VERSION}/pgbouncer-${PGBOUNCER_VERSION}.tar.gz
```

Comment:
@beanieboi Hi :-) After this change the download URL isn't using HTTPS. Checking it seems that the host supports it?
eg: https://www.pgbouncer.org/downloads/files/1.13.0/pgbouncer-1.13.0.tar.gz

Reply:
thx, fixed it

Reply:
Thank you :-)